projects/packages/assets/tests/php/test-assets.php

```diff
@@ -50,9 +51,9 @@ public function set_up() {
 					return $url;
 				},
 				'plugins_url'    => function ( $path, $plugin_path ) use ( $plugin_dir ) {
-					$plugin_path = dirname( $plugin_path );
-					$this->stringStartsWith( $plugin_dir, $plugin_path );
-					return 'http://www.example.com/wp-content/plugins/jetpack/' . substr( $plugin_path, strlen( $plugin_dir ) ) . '/' . $path;
+					$plugin_path = dirname( $plugin_path ) . '/';
+					$this->assertStringStartsWith( $plugin_dir, $plugin_path );
+					return 'http://www.example.com/wp-content/plugins/jetpack/' . substr( $plugin_path, strlen( $plugin_dir ) ) . $path;
```

Comment:
Is a trailing slash a Phan rule?

Reply:
So the old code wasn't actually testing anything. `$this->stringStartsWith()` creates a Constraint object, which you'd have to pass to `$this->assertThat()` (or make calls on it directly) in order to actually test anything. But the code here just threw it away.

Obviously it meant to be doing `->assertStringStartsWith()`. But when I added that I found some of the tests failed because `$plugin_dir` was `/path/to/something/or/other/` while `$plugin_path` was just `/path/to/something/or/other`. Adding the trailing slash to `$plugin_path` fixed that.